scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/ScriptData.java

```diff
@@ -46,7 +46,7 @@
 
 
 @Entity
-@NamedQueries({ @NamedQuery(name = "deleteScriptData", query = "delete from ScriptData where taskData.id.jobId = :jobId"),
+@NamedQueries({ @NamedQuery(name = "deleteScriptData", query = "delete from ScriptData where taskData.id.jobId in :ids"),
```

Comment:
don't we have already the deleteScriptDataInBulk for this purpose ?

Reply:
you are right